Dockerfile

```diff
@@ -1,5 +1,5 @@
 # For more information, please refer to https://aka.ms/vscode-docker-python
-FROM python:3.8-slim

```

Comment:
I think we should stick to Python 3.8 -- Beanstalk does not support higher versions, but even though we are not planning to deploy the composite service on Beanstalk, it might be safer to stick to 3.8.

Reply:
Reverted to 3.8!